invenio_administration/assets/semantic-ui/js/invenio_administration/details/DetailsActions.js

```diff
@@ -0,0 +1,128 @@
+// This file is part of InvenioAdministration
+// Copyright (C) 2022 CERN.
+//
+// Invenio RDM Records is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import PropTypes from "prop-types";
+import React, { Component } from "react";
+import { i18next } from "@translations/invenio_administration/i18next";
+import { Segment, Header, Divider, Button } from "semantic-ui-react";
+import ActionModal from "./ActionModal";
+import { Delete } from "../actions/Delete";
+import { Edit } from "../actions/Edit";
+
+class DetailsActions extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      modalOpen: {},
```

Comment:
why not boolean but empty dict?

Reply:
Because it's used to track the state of the different modals. There can be more than just one, as you can see in the template.